playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,145 @@
+- IAM permissions for Cloud Build service account - `playground-cloudbuild-sa`
+
+#### To execute the module:
+
+1. Set following environment variables:
+   - `STATE_BUCKET`: GCP Storage bucket name to save Terraform state
+   - `GOOGLE_PROJECT`: GCP Project ID
+   - `GOOGLE_REGION`: GCP region to save resources to
+   - `ENVIRONMENT_NAME`: Environment where Playground will be deployed (located at playground/terraform/environment/environment_name)
+   - `DNS_NAME`: DNS for deployed Playground webpage
+   - `NETWORK_NAME`: GCP VPC Network Name for Playground deployment
+   - `GKE_NAME`: Playground GKE Cluster name
+   - `TAG`: Tag for Playground images
+   - `DOCKER_REPOSITORY_ROOT`: GCP Artifact Registry repository name to store Playground container images
+
+```console
+    export STATE_BUCKET="state_bucket" \
+    GOOGLE_PROJECT="project_id" \
+    GOOGLE_REGION="region" \
+    ENVIRONMENT_NAME="env_name" \
+    DNS_NAME="dns_name" \
+    NETWORK_NAME="network_name" \
+    GKE_NAME="gke_name" \
+    TAG="tag" \
+    DOCKER_REPOSITORY_ROOT="us-central1-docker.pkg.dev/playground-repository"
+```
+**Note:**  Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations)     for the list of all supported locations.
+
+2. Run commands:
+
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="bucket=$STATE_BUCKET"
+terraform apply -var "project_id=$GOOGLE_PROJECT"
+```
+
+## 2. Connect Apache Beam GitHub repository and GCP Cloud Build
+
+**Note:** Ensure correct `region` is set in [Cloud Build Triggers](https://console.cloud.google.com/cloud-build/triggers) page before proceeding further.
+
+Follow [Connect to a GitHub repository](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github) to connect Apache Beam GitHub repository and GCP Cloud Build.
+
+## 3. Set up the Google Cloud Build triggers
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders` provisions:
+- Cloud Build triggers to build and deploy Beam Playground
+
+#### To execute the module
+
+
+```
+# Navigate to playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders folder
+cd ../02.builders
+
+# Run terraform scripts
+terraform init -backend-config="bucket=$STATE_BUCKET"
+
+terraform apply \
+-var "project_id=$GOOGLE_PROJECT" \
+-var "region=$GOOGLE_REGION" \
+-var "playground_environment_name=$ENVIRONMENT_NAME" \
+-var "playground_dns_name=$DNS_NAME" \
+-var "playground_network_name=$NETWORK_NAME" \
+-var "playground_gke_name=$GKE_NAME" \
+-var "state_bucket=$STATE_BUCKET" \
+-var "image_tag=$TAG" \
+-var "docker_repository_root=$DOCKER_REPOSITORY_ROOT"
```

Comment:
var.cloudbuild_service_account_id
  The ID of the cloud build service account responsible for provisioning Google Cloud resources

Reply:
Latest changes pull required